packages/editor/src/client/app.tsx

```diff
@@ -70,6 +70,7 @@ export function contentForRoute(route: Route) {
     case RouteType.UserList:
     case RouteType.UserCreate:
     case RouteType.UserEdit:
+    case RouteType.UserExport:
```

Comment:
there is no need for a new route. You can just open a new modal in the userlist. Similar to the preview link modal in the articlesList.